src/main/kotlin/edu/cwru/raycaster/Raycaster.kt

```diff
@@ -55,13 +57,16 @@ fun stringToBlockMap(str: String): Array<Array<Block>> {
 }
 
 private val MAP = stringToBlockMap("""
-    ######
-    #    #
-    #  # #
-    #    #
-    ######
+    BBBBBB
+    G    B
+    B  B B
+    O    B
+    BBBBBB
 """.trimIndent())
 
+val SKY_COLOR: Color = Color.LIGHTBLUE
```

Comment:
```suggestion
val SKY_COLOR = Color.LIGHTBLUE
```

Reply:
IntelliJ gets angry because it can't be sure that Color.LIGHTBLUE isn't null and wants a specific type annotation